www/include/options/session/connected_user.php

```diff
@@ -37,18 +37,92 @@
     exit();
 }
 
-$path = "./include/options/session/";
+/**
+ * Kick a logged user
+ */
+const KICK_USER = "k";
+
+/**
+ *  Manually plan the synchronization of the chosen user on next login
+ */
+const SYNC_USER = "s";
 
+$path = "./include/options/session/";
 require_once "./include/common/common-Func.php";
 require_once "./class/centreonMsg.class.php";
 
-if (isset($_GET["o"]) && $_GET["o"] == "k") {
-    $sid = $_GET['session'];
-    $pearDB->query("DELETE FROM session WHERE session_id = '" . $pearDB->escape($sid) . "'");
+$action = filter_var(
+    $_GET["o"] ?? null,
+    FILTER_SANITIZE_STRING
+);
+
+$selectedUserSid = filter_var(
+    $_GET['session'] ?? null, // the sessionId of the chosen user
+    FILTER_SANITIZE_STRING
+);
+
+$currentPage = filter_var(
+    $_GET['p'] ?? $_POST['p'] ?? 0,
+    FILTER_VALIDATE_INT
+);
+
+if ($selectedUserSid) {
     $msg = new CentreonMsg();
     $msg->setTextStyle("bold");
-    $msg->setText(_("User kicked"));
     $msg->setTimeOut("3");
+
+    switch ($action) {
```

Comment:
no default action ?

Reply:
no. The default action, is to display the list